src/views/components/admin/lotsList.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+import { Alert } from 'react-bootstrap';
+import useGetParkingLots from '../../../controllers/useGetParkingLot.js';
+import ParkingLot from '../admin/parkingLotCard.js';
+
+function ParkingLotsList() {
+    const {  parkingLots, fetchParkLots, error } = useGetParkingLots();
```

Comment:
useGetParkingLots calls the `/driver-parking-lots` endpoint - which is meant for drivers. You shold make another model that calls this endpoint https://github.com/IgorSteps/easypark/blob/main/docs/API_SPEC.md#2-get-all-parking-lots-api-endpoint

Reply:
<img width="654" alt="Screenshot 2024-05-15 at 13 37 58" src="https://github.com/IgorSteps/easypark-ui/assets/84919282/2920e633-c590-44d3-a4f8-6a7d4a67ce53">
Like the logs tell you in the backend